### PR TITLE
add bundle install instructions

### PR DESCRIPTION
**Pull request by sofroniewn**
Add install instructions for bundled app, with details on mac process, including screenshots

@jni @tlambert03 can you review this - feel free to rip it apart/ rewrite it, i'll be honest i was pretty rushed getting this done, but i wanted to have something in before tweeting out the bundled app as i know so many people will run into this problem on mac

feel free to add windows/ linux details too. i'd say goal right now isn't to be perfect, but to be better than nothing!!

### CODE REVIEW

File: fundamentals/installation.md [hunk 1]
```diff
@@ -96,6 +96,62 @@ pip install napari[pyqt5]
...
+
+![image]({{ '/assets/tutorials/installation/bundle_applications_folder.png' | relative_url }})
+
+The first time you try and open napari you will get the "unverified developer warning" that
+appears bellow:
+
+![image]({{ '/assets/tutorials/installation/bundle_dev_warning.png' | relative_url }})
+
+Don't be alarmed, this is because the napari team has not yet obtained a developer certificate
+from Apple, but we will be doing this promtly. It is safe to click `Cancel`. To get napari to
```
Comment (tlambert03) [inline fundamentals/installation.md:136]:
```suggestion
from Apple, but we will be doing this promptly. It is safe to click `Cancel`. To get napari to
```

File: fundamentals/installation.md [hunk 2]
```diff
@@ -96,6 +96,62 @@ pip install napari[pyqt5]
...
+open you must enter `Systems Preferences` and click the `Security & Privacy` icon, circled in
+red below:
+
+![image]({{ '/assets/tutorials/installation/bundle_security_privacy.png' | relative_url }})
+
+Once inside the `Security & Privacy` tab you should see a message about napari being blocked
```
Comment (tlambert03) [inline fundamentals/installation.md:142]:
```suggestion
Once inside the `Security & Privacy` tab, you should see a message about napari being blocked
```

File: fundamentals/installation.md [hunk 3]
```diff
@@ -96,6 +96,62 @@ pip install napari[pyqt5]
...
+and an `Open Anyway` button, circled in red below:
```
Comment (tlambert03) [inline fundamentals/installation.md:143]:
```suggestion
and an "Open Anyway" button, circled in red below:
```

File: fundamentals/installation.md [hunk 4]
```diff
@@ -96,6 +96,62 @@ pip install napari[pyqt5]
...
+
+![image]({{ '/assets/tutorials/installation/bundle_open_anyway.png' | relative_url }})
+
+After clicking this button you'll get one final warning about napari not being verified, but
+now you'll have an `Open` button, as seen below:
```
Comment (tlambert03) [inline fundamentals/installation.md:148]:
```suggestion
now you'll have an "Open" button, as seen below:
```

File: fundamentals/installation.md [hunk 5]
```diff
@@ -96,6 +96,62 @@ pip install napari[pyqt5]
...
+
+![image]({{ '/assets/tutorials/installation/bundle_open.png' | relative_url }})
+
+Now you'll finally see the napari viewer appear, but don't worry, you only have to go through
```
Comment (tlambert03) [inline fundamentals/installation.md:152]:
```suggestion
After clicking "Open", the viewer should appear.  Don't worry, you will only have to go through
```

**Review by tlambert03 — APPROVED** [review]
looks good to me, made some minor typos and punctuations suggestions

**Comment by jni** [comment]
@sofroniewn one change I took some liberty with is adding section placeholders for Windows and Linux. I thought it better than to not even acknowledge those users. Let me know if you disagree with those changes and I can submit a PR to revert them.

**Comment by GenevieveBuckley** [comment]
> 
> 
> @sofroniewn one change I took some liberty with is adding section placeholders for Windows and Linux. I thought it better than to not even acknowledge those users. Let me know if you disagree with those changes and I can submit a PR to revert them.

I think this is a good idea too. Even if the content ends up being "this should be easy on your platform" it's *much* more reassuring to read that as a new user than have to guess.

**Comment by GenevieveBuckley** [comment]
I can maybe put something together for Windows, and perhaps Juan can do the one for Linux?

**Comment by sofroniewn** [comment]
> I can maybe put something together for Windows, and perhaps Juan can do the one for Linux?

That would be great, thanks!!

**Comment by jni** [comment]
> I can maybe put something together for Windows, and perhaps Juan can do the one for Linux?

Yes, that would be awesome. I can do the Linux guide later today.

**Comment by GenevieveBuckley** [comment]
@sofroniewn it doesn't look like the markdown images are being rendered correctly?

If I look [here on your branch](https://github.com/sofroniewn/napari-tutorials/blob/bundled_app_install/fundamentals/installation.md) then I see this instead of an image following the text:
```
An empty napari viewer should appear as follows

![image]({{ '/assets/tutorials/launch_cli_empty.png' | relative_url }})
```

**Pull request merged**